<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cluster</class>
 <widget class="QMainWindow" name="Cluster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cluster</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="motor_temp">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="high_batt_temp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="mtr_cntrl_temp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
